source/components/rp2017/speakers/speakers.js

```diff
@@ -13,6 +13,12 @@ class Speakers extends Component {
 
         const speakerList = [
              // Keynotes
+             {
+                'name': 'Siraj Raval',
+                'img': '../assets/img/keynotes/siraj.jpg',
+                'year': 'YouTuber - Accessible CS education',
+                'wikipedia': 'https://twitter.com/sirajraval?lang=en'
```

Comment:
@modnikhil Are we OK with non-Wikipedia links?

Reply:
Should be fine as long as we specifically don't say "wikipedia page"